src/components/KMultiselect/KMultiselect.vue

```diff
@@ -1108,32 +1139,33 @@ onBeforeUnmount(() => {
 
   .k-multiselect-popover {
     box-sizing: border-box;
-    margin-top: 2px !important;
+    margin-top: var(--kui-space-0, $kui-space-0) !important;
+    margin-top: var(--kui-space-10, $kui-space-10) !important;
     width: 100%;
 
     &[x-placement^="top"] {
-      margin-bottom: 2px !important;
-      margin-top: 0 !important;
+      margin-bottom: var(--kui-space-10, $kui-space-10) !important;
+      margin-top: var(--kui-space-0, $kui-space-0) !important;
     }
 
     &.k-multiselect-pop {
-      --KPopPaddingY: var(--spacing-xs);
-      --KPopPaddingX: var(--spacing-xs);
-      border: 1px solid var(--black-10);
+      --KPopPaddingY: var(--spacing-xs, var(--kui-space-40, #{$kui-space-40}));
+      --KPopPaddingX: var(--spacing-xs, var(--kui-space-40, #{$kui-space-40}));
+      border: var(--kui-border-width-10, $kui-border-width-10) solid $tmp-color-black-10;
```

Comment:
question: can you explain the need for `$tmp-color-black-10`? What happens to these when we do the big clean up effort to remove refs to the previous color vars? Or is this like a TODO saying we need this color to be a token?